pkg/apperror/error.go

```diff
@@ -29,19 +30,19 @@ func New(code Code, err error) appError {
 	}
 }
 
-func NewGorm(err error) appError {
+func NewGorm(err error, msg string) appError {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported func NewGorm returns unexported type github.com/ispec-inc/go-distributed-monolith/pkg/apperror.appError, which can be annoying to use (golint)